src/common/config/lorem-ipsum-options.ts

```diff
@@ -0,0 +1,9 @@
+export interface LoremIpsumOptions {
+    isEnabled: boolean;
+    prefix: string;
+}
+
+export const defaultLoremIpsumOptions: LoremIpsumOptions = {
+    isEnabled: true,
```

Comment:
As this is a plugin for very specific users, I suggest to make it disabled by default.

Reply:
> Thank you for your contribution! I left some comments.

Everything you commented should be fixed now. Thanks for the fast reply